tests/ztest/ztress/prj.conf

```diff
@@ -1,2 +1,4 @@
 CONFIG_ZTEST=y
 CONFIG_ZTRESS=y
+CONFIG_SMP=n
```

Comment:
Hmm,

as I remember from previous discussions it's not correct to disable SMP from the test case.
We should either set `CONFIG_MP_NUM_CPUS=1` (so it would be SMP system with only one cpu core) or use special macroses like `ztest_1cpu_unit_test` to run test case on 1 cpu.

Reply:
done